journal-parsing/src/main/java/com/fnz/db2/journal/retrieve/RetrieveJournal.java

```diff
@@ -449,10 +449,10 @@ public ParameterListBuilder filterJournalCodes(JournalCode[] codes) {
 		    return this;
 		}
 
-	      public ParameterListBuilder filterFiles(List<String> includeFiles) {
-	            criteria.addFILE(this.receiverLibrary, includeFiles);
```

Comment:
It occurs to me now we could probably have simply changed `this.receiverLibrary` to be the schema...